src/Blast.jl

```diff
@@ -7,9 +7,34 @@ using FastChebInterp
 using SpecialFunctions
 using StaticArrays
 using FFTW
+using NPZ
+using Artifacts
 
 include("projected_matter.jl")
 include("chebcoefs.jl")
 include("integrals.jl")
 
+function load_precomputed_Ts(folder; nℓ=22, nχ=96, nR=48, n_cheb=120)
+    ell_vector =reverse(chebpoints(100, 2,2000))[1:nℓ]
+    full_T = zeros(nℓ, nχ, nR, n_cheb)
+    for i in 1:nℓ
+        l_string = string(round(ell_vector[i]; digits=1))
+        filename = joinpath(folder, "T_tilde_l_$l_string.npy")
+        if isfile(filename)
+            full_T[i,:,:,:] = npzread(filename)
+        else
+            println("Missing file!")
```

Comment:
Although it is fine to print a message in this way, it might not be seen from the user.
We might want to be throwing an `error` in this [case](https://docs.julialang.org/en/v1/base/base/#Base.error).